lib/node_modules/@stdlib/constants/float32/half-pi/README.md

```diff
@@ -0,0 +1,147 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# HALF_PI
+
+> One half times the mathematical constant [π][pi].
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var HALF_PI = require( '@stdlib/constants/float32/half-pi' );
+```
+
+#### HALF_PI
+
+One half times the mathematical constant [π][pi].
+
+```javascript
+var bool = ( HALF_PI === 1.5707963 );
```

Comment:
This doesn't appear to be to "full" double-precision. Notice the difference below:

```
In [13]: new Float32Array( [ HALF_PI ] )[ 0 ]
Out[13]: 1.5707963705062866

In [14]: HALF_PI
Out[14]: 1.5707963267948966
```

Here, we are doing the comparison in double-precision, so use the full precision result.